src/constants.js

```diff
@@ -19,6 +21,11 @@ export const RELEASES = new Map([
   [CENTOS_8, 'CentOS Stream 8'],
 ]);
 
+export const ARCHS = new Map([
+  [X86_64, 'x86'],
```

Comment:
```suggestion
  [X86_64, 'x86_64'],
```

I suggest being explicit here. :)

Reply:
`x86` is conform to Andy's mock, see https://www.sketch.com/s/d7aa6d29-fca0-4283-a846-09cc5fd10612/a/My1W1mm

Reply:
Totally makes sense to build it to the mocks, but one thing to keep in mind is that the designers lean on us as subject matter experts... Design is a team effort. :wink: 

Let's change to `x86_64`. `x86_64` also appears in places like the request, which users will encounter if they are using the API or choose to download the request .json file, so this way it will be consistent.

Reply:
Done.